### PR TITLE
net-dns/getdns: fix forgot use_with replaced by usex and switch to cmake.eclass

### PR DESCRIPTION
**Pull request by CaseOf**
Forgot this one, now fixed

### CODE REVIEW

File: net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+IUSE="doc examples +getdns-query +getdns-server-mon gnutls +idn libev libevent libuv static-libs stubby +unbound"
+
+S="${WORKDIR}/${_SRCURI_P}"
+
+# https://bugs.gentoo.org/661760
+# https://github.com/getdnsapi/getdns/issues/407
+RESTRICT="test"
+
+DEPEND="
+	dev-libs/libbsd:=
```
Comment (Zlogene) [inline net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild]: there is no subslot, so `:=` is not needed

File: net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	dev-libs/libyaml:=
```
Comment (Zlogene) [inline net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild]: the same.

File: net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+	dev-libs/openssl:0=
```
Comment (Zlogene) [inline net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild]: you do not need to specify `0`

File: net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+	libevent? ( dev-libs/libevent:= )
+	libuv? ( dev-libs/libuv:= )
+	unbound? ( >=net-dns/unbound-1.5.9:= )
+"
+RDEPEND="
+	${DEPEND}
+	stubby? (
+		acct-group/stubby
+		acct-user/stubby
+		sys-libs/libcap:=
```
Comment (Zlogene) [inline net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild]: the same.

File: net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+		-DUSE_GNUTLS=$(usex gnutls)
+		-DUSE_LIBIDN2=$(usex idn)
+		-DUSE_LIBEV=$(usex libev)
+		-DUSE_LIBEVENT2=$(usex libevent)
+		-DUSE_LIBUV=$(usex libuv)
+		-DBUILD_STUBBY=$(usex stubby)
+		-DENABLE_UNBOUND_EVENT_API=$(usex unbound)
+		-DBUILD_DOXYGEN=$(usex doc)
+	)
+	cmake-utils_src_configure
```
Comment (Zlogene) [inline net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild]: you havr no `cmake-utils` byt only `cmake` eclass

File: net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+src_install() {
+	cmake-utils_src_install
```
Comment (Zlogene) [inline net-dns/getdns/getdns-1.6.0_beta1-r1.ebuild]: the same.

**Comment by CaseOf** [comment]
ping @blueness

**Comment by blueness** [comment]
is this ready to go?

**Comment by CaseOf** [comment]
> is this ready to go?

Yes

**Comment by blueness** [comment]
pushed

**Pull request closed without merging**